apps/src/lib/kits/maker/microBitRedux.js

```diff
@@ -0,0 +1,21 @@
+const SET_MICROBIT_FIRMATA_UPDATE_PERCENT =
+  'microBit/SET_MICROBIT_FIRMATA_UPDATE_PERCENT';
+
+const initialState = {
+  microBitFirmataUpdatePercent: null
```

Comment:
Minor - I might initialize this to 0 rather than null, so that the view is always dealing with a number (e.g. when checking percent > 0)